src/bentoml/_internal/frameworks/diffusers.py

```diff
@@ -45,12 +47,15 @@
 
 logger = logging.getLogger(__name__)
 
+LoraOptionType = str | dict[str, str]
+TextualInversionOptionType = str | dict[str, str]
```

Comment:
```suggestion
LoraOptionType = t.Union[str, t.Dict[str, str]]
TextualInversionOptionType = t.Union[str, t.Dict[str, str]]
```

Since this type is using for ModelOptions and will be serialized, I think we need to use this instead of the previous notation for python <3.10